cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -222,28 +223,28 @@ public String toString() {
                     .withDeleteClaim(true)
                     .build()
         };
-        Map[] kafkaConfigs = {
+        List<Map<String, Object>> kafkaConfigs = asList(
             null,
             emptyMap(),
             singletonMap("foo", "bar")
-        };
```

Comment:
Attempting to remove raw types warning. Used `asList` instead of `List.of` as the latter's arguments cannot be null.